### PR TITLE
use py38 base img

### PR DESCRIPTION
**Pull request by chrieke**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM up42/up42-snap-py38:latest
```
Comment (markusuwe) [inline Dockerfile:1]: Why is the version tag not needed any more?

**Review by markusuwe — APPROVED** [review]
LGTM

**Review by marooned20 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (chrieke) [reply Dockerfile:1]: Not required since we always only have a single version in dockerhub anyway. Can add it again if prefered?

Comment (markusuwe) [reply Dockerfile:1]: No, please remove. Was just curious.

**Pull request merged**